logic/labrynna.go

```diff
@@ -158,13 +163,16 @@ var labrynnaNodes = map[string]*Node{
 	"ridge base past east": Or("target carts",
 		And("lynna city", Or("feather", "ages"), "mermaid suit"),
 		And("ridge mid past", "feather", "brother emblem"),
-		And("ridge mid present", "ages"),
+		"rolling ridge east tree",
+		And("ridge base present", "ages"),
 		And("ridge base past west", Or("flippers", Hard("jump 3")))),
 	"ridge base past west": Or(
 		And("ridge base present", "echoes"),
-		And("ridge base past east", Or("flippers", Hard("jump 3")))),
+		And("ridge base past east", Or("flippers", "bomb jump 2"))),
 	"ridge base past":     AndSlot("ridge base past west", "bombs"),
-	"enter d6 past":       And("mermaid key", "ridge base past west"),
+	"enter d6 past":       And("mermaid key", Or(
+		And("ridge base past west", Or("flippers", "feather", "ages")),
+		And("ridge base past east", "bomb jump 2"))),
```

Comment:
The reason "ridge base past east" isn't included here is that if you can get from RBPW to D6 past, you can also get there from RBPE. Amazing that it was missing any actual entry requirements, though.

Reply:
So this should read: 
```go
"enter d6 past": And("mermaid key", "ridge base past west",
    Or("flippers", "ages", Hard("bomb jump 2")),
```